rb/lib/htmlproofer/enforce_img_https.rb

```diff
@@ -0,0 +1,17 @@
+class EnforceImgHTTPS < ::HTMLProofer::Check
+  def run
+    @html.css('img').each do |node|
+      img = create_element(node)
```

Comment:
I agree we don't need to support them, I do worry (perhaps without enough justification) that if someone tries using them, possibly inadvertently, they'll waste time wondering why the tests failed.